sdk/remoterendering/azure-mixedreality-remoterendering/tests/test_client_async.py

```diff
@@ -210,8 +210,10 @@ async def test_failed_conversion_missing_asset(self, recorded_test, account_info
             await conversion_poller.result()
 
         error_details = excinfo.value
-        assert "invalid input" in error_details.error.message.lower()
-        assert "logs" in error_details.error.message.lower()
+        assert "InputContainerError" == error_details.error.code
+        # Message: "Could not find the asset file in the storage account. Please make sure all paths and names are correct and the file is uploaded to storage."
+        assert None != error_details.error.message
+        assert "Could not find the asset file in the storage account" in error_details.error.message
```

Comment:
Is this message always in English? Is there any case it may be in non-English langauges?

Reply:
Yes this message is always in English.